frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -315,27 +322,41 @@ function VirtualKeyPopup:init()
     local parent_key = self.parent_key
     local key_chars = parent_key.key_chars
     local key_char_orig = key_chars[1]
+    local key_char_orig_func = parent_key.callback
```

Comment:
What's that for?

Reply:
The middle button responds like the original key so I figure it should do the same with a custom callback. Could also just be nil I figure.

Reply:
Oh, allright, I thought you were saving something before allowing/going to override it (that I often would name `something_orig_func` :)